sui/wormhole/Move.lock

```diff
@@ -2,19 +2,26 @@
 
 [move]
 version = 0
+manifest_digest = "20C2ADF9411233240C3FA257B9A558910826BB5EBBCDB54F0D0816D61333B4A8"
+deps_digest = "F8BBB0CCB2491CA29A3DF03D6F92277A4F3574266507ACD77214D37ECA3F3082"
 
 dependencies = [
   { name = "Sui" },
 ]
 
 [[move.package]]
 name = "MoveStdlib"
-source = { git = "https://github.com/MystenLabs/sui.git", rev = "09b2081498366df936abae26eea4b2d5cafb2788", subdir = "crates/sui-framework/packages/move-stdlib" }
+source = { git = "https://github.com/MystenLabs/sui.git", rev = "framework/testnet", subdir = "crates/sui-framework/packages/move-stdlib" }
```

Comment:
might be best to keep these dependencies pinned to a commit hash

Reply:
We have to take into account that `Sui` (along with `MoveStdlib` at "0x1"
AFAIK) is a special package which can be upgraded onchain 'in-place'
(unlike regular Move package upgrades on Sui that in effect create a new
package with a new address.)

The advantage of using `framework/testnet` is that the compiler will
automatically fetch the `Sui` package that's currently on testnet, hence
source verification will succeed.
```
$ sui client verify-source --verify-deps
UPDATING GIT DEPENDENCY https://github.com/MystenLabs/sui.git
INCLUDING DEPENDENCY Sui
INCLUDING DEPENDENCY MoveStdlib
BUILDING Wormhole
// ...
Source verification succeeded!
```

However, let's say we published the package in the past and set `Sui`
to `testnet-v1.17.1` in the manifest. Verification could fail in the
future to to upgrades to the `Sui` package.
```
$ sui client verify-source --verify-deps
FETCHING GIT DEPENDENCY https://github.com/MystenLabs/sui.git
INCLUDING DEPENDENCY Sui
INCLUDING DEPENDENCY MoveStdlib
BUILDING Wormhole
// ...
Multiple source verification errors found:

- Local dependency did not match its on-chain version at 0000000000000000000000000000000000000000000000000000000000000002::Sui::bls12381
- Local version of dependency 0000000000000000000000000000000000000000000000000000000000000002::group_ops was not found.
```

For all other packages, it definitely makes sense to pin them to a
commit/tag.